WalletWasabi.Fluent/Views/Settings/GeneralSettingsTabView.axaml

```diff
@@ -26,10 +26,15 @@
     </DockPanel>
 
     <DockPanel>
-      <TextBlock Text="Auto Copy/Paste Bitcoin addresses" />
+      <TextBlock Text="Auto Copy receive addresses" />
       <ToggleSwitch IsChecked="{Binding AutoCopy}" />
     </DockPanel>
 
+    <DockPanel>
+      <TextBlock Text="Auto Paste send addresses" />
```

Comment:
Saying `receive/send addresses` is confusing for me. I would say `Auto Paste "receive" addresses` makes more sense than `Auto Paste "send" addresses` because we paste the address that receives the bitcoin not the one that sends it.

So my suggestion is to simply remove the words `receive` and `send`:

```suggestion
      <TextBlock Text="Auto copy addresses" />
      <ToggleSwitch IsChecked="{Binding AutoCopy}" />
    </DockPanel>

    <DockPanel>
      <TextBlock Text="Auto paste addresses" />
```

Reply:
ACK